### DISCUSSION TITLE
How do you connect to other EVM blockchains like Celo

### DISCUSSION THREAD

**Discussion by ohenepee** [WalletConnect/walletconnect-monorepo#1769] [Developer Support]
I've spent the last 5 hours searching here and there both on WalletConnect v2 & Web3Modal pages for using custom chains, there's guide on that except [here](https://docs.walletconnect.com/2.0/web3modal/configuration) where it mentions `standaloneChains` with some short text and an example that has literally no meaning for anyone new to WallectConnect or Web3Modal.

**Comment by chadyj** [comment]
Celo is an example in our sample app
See here https://react-app.walletconnect.com/

Code and more can be found here https://github.com/WalletConnect/web-examples

**Reply by ohenepee** [reply]
It does not address connecting to custom EVM chains.

**Reply by aaronmgdr** [reply]
celo and other evm chains can be used by following https://docs.walletconnect.com/2.0/web3modal/react/installation

and in the line `import { arbitrum, mainnet, polygon } from "wagmi/chains";` replace with the chains you want such as `import { celo, celoAlfajores, polygon } from "wagmi/chains";

then construct your chains array. 


I just did this in a little demo and it works
